core/modules/user/user.theme.inc

```diff
@@ -172,15 +172,18 @@ function theme_user_permission_description($variables) {
 function theme_user_admin_roles($variables) {
   $form = $variables['form'];
 
-  $header = array(t('Name'), t('Weight'), t('Operations'));
+  $header = array(t('Name'), t('Status'), t('Weight'), t('Operations'));
   foreach (element_children($form['roles']) as $role_name) {
-    $label = $form['roles'][$role_name]['#role']->label;
     $row = array();
+    $row[] = theme('label_machine_name__role', array(
+      'label' => $form['roles'][$role_name]['#role']->label,
+      'machine_name' => $form['roles'][$role_name]['#role']->name,
+    ));
     if (in_array($role_name, array(BACKDROP_ANONYMOUS_ROLE, BACKDROP_AUTHENTICATED_ROLE))) {
-      $row[] = t('@name <em>(required)</em>', array('@name' => $label));
+      $row[] = '<em>' . t('Required') . '</em>';
```

Comment:
Lets not emphasize this:

```suggestion
      $row[] = '<em>' . t('required') . '</em>';
```

Reply:
Do you mean 'emphasize' or 'capitalise'...?